lib/engine/step/g_18_sj/special_buy.rb

```diff
@@ -27,7 +27,7 @@ def process_special_buy(action)
 
         def setup
           super
-          @gkb_bonus_item = Item.new(description: 'GKB bonus', cost: 0)
+          @gkb_bonus_item ||= Item.new(description: 'GKB bonus', cost: 0)
```

Comment:
why is this changed?

Reply:
This was supposed to be in the prior PR but the push from my working copy failed due to a typo

Reply:
is this needed?

Reply:
It will work as-is. This change will prevent a new instance of the Item being created every operating round